docs/app/pages/components/components-sections/timeline/timeline/snippets/app.html

```diff
@@ -2,21 +2,23 @@
     <div class="col-md-10">
         <button class="btn button-primary" (click)="addEvent()">Add New Event</button>
         <ux-timeline>
-            <ux-timeline-event *ngFor="let event of events" [badgeColor]="event.color"
-                [badgeTitle]="event.date | date:'EEE LLL d'">
+            <li *ngFor="let event of events">
```

Comment:
Why is this li element here? Can the ngFor not just be on the ux-timeline-event element?

Reply:
It seems quite messy and an unnecessary complexity to have to have them insert an li around each event. 

If we take a look at all other libraries like ngx-bootstrap where they have a similar structure eg. accordion, tabs etc.. they don't require anything like this. If the UX timeline component is currently using a UL then perhaps it should just be changed to use a div or something instead removing the requirement for the LI.

Reply:
The 'li' element was introduced becuae of the existing 'ul' element. I will look into replacing the 'ul' as you suggested.